packages/phone-number-privacy/src/salt-generation/bls-cryptography-client.ts

```diff
@@ -0,0 +1,40 @@
+import { AzureKeyVaultClient } from '@celo/contractkit/src/utils/azure-key-vault-client'
+import { BLINDBLS } from 'bls12377js-blind'
+import config from '../config'
+
+export class BLSCryptographyClient {
```

Comment:
Is there a particular reason for making this a class? Like does it have some state it should encapsulate.
This is more of a style point but it's not very Javascripty to create a class and then fill it with static functions. There's no real downside to just putting those functions at the root scope and then exporting them. Node's module system (a file is a module) will already encapsulate any variables you declare in this scope

Reply:
I'm new to nodeJS so I'm probably just falling back to my C# habits 😄 I made it a class since it has the `privateKey` state which should be a private static property of the class. Is it recommended to just have this in a module?

Reply:
Spoke offline; due to a limitation with Jest, we'll lose the ability to dynamically mock (test against mock and actual KV) if we remove the class and make it a module. Will keep as is.